### PR TITLE
Account RAF Tab FAQ Section

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request adds an FAQ section to the RAF account tab.

### How should this be reviewed?
👀  Are the Detail/Summaries styled properly?

### Any background context you want to provide?
We're using the same designs as existing detail/summary support via Contentful fields / Markdown [here](https://github.com/DoSomething/phoenix-next/blob/b9a2703b6080f01f07b8e44bf43b52c3b13979c8/resources/assets/components/utilities/TextContent/markdown.scss#L137-L149). 

It should be easy to extract these styled-components out into their own utilities if need be.

### Relevant tickets

References [Pivotal #172748957](https://www.pivotaltracker.com/story/show/172748957).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [x] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

image

- [LARGE EXPANDED](https://user-images.githubusercontent.com/12417657/85421792-d5f08480-b542-11ea-956d-4088d0ca637b.png)
- [MEDIUM COLLAPSED](https://user-images.githubusercontent.com/12417657/85421833-e30d7380-b542-11ea-9919-5e9de2276c5c.png)
- [MEDIUM EXPANDED](https://user-images.githubusercontent.com/12417657/85421860-ebfe4500-b542-11ea-8e19-5f38dc36d157.png)
- [SMALL COLLAPSED](https://user-images.githubusercontent.com/12417657/85421885-f3bde980-b542-11ea-85d7-234896ff4d26.png)
- [SMALL EXPANDED](https://user-images.githubusercontent.com/12417657/85421907-fae4f780-b542-11ea-8408-7596faafa33f.png)

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js [hunk 1]
```diff
@@ -19,9 +24,45 @@ const ReferFriendsTab = () => (
       />
     </div>
 
-    <div className="col-span-4 md:col-span-8 lg:col-start-2 lg:col-span-11 xxl:col-start-2 xxl:col-span-10">
+    <div className="col-span-4 md:col-span-8 lg:col-start-2 lg:col-span-11 xxl:col-start-2 xxl:col-span-10 pt-4">
       <SignupReferralsGallery />
     </div>
+
+    <div className="col-span-4 md:col-span-8 lg:col-start-2 lg:col-span-7 xxl:col-start-2 xxl:col-span-6 pt-10">
```
Comment (mendelB) [inline resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js:31]: It certainly might be high time to extract some utility components or macros encapsulating these grid classes, but I still don't have any precise direction on this front to confidentially formalize it. (This is kinda my own ad hoc based on existing grid utility classes and the designs).

Comment (weerd) [reply resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js:31]: Ya potentially as long as there's a distinct repeated pattern. I can help with this a bit after my Algolia work if this isn't pressing for now.

Comment (mendelB) [reply resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js:31]: Thanks @weerd, that would be awesome! (And not pressing for now).

File: resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import tw from 'twin.macro';
 
 import SignupReferralsGallery from './SignupReferralsGallery';
 import SectionHeader from '../../../utilities/SectionHeader/SectionHeader';
 import SocialDriveActionContainer from '../../../actions/SocialDriveAction/SocialDriveActionContainer';
 
+const Details = tw.details`pb-4`;
+const Summary = tw.summary`font-bold text-base cursor-pointer`;
+const DetailsParagraph = tw.p`pt-2`;
```
Comment (aaronschachter) [inline resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js:10]: This is so neat! I've never seen/noticed this in the codebase before.

Comment (mendelB) [reply resources/assets/components/pages/AccountPage/ReferFriends/ReferFriendsTab.js:10]: Right?! Dave added the support in #1965 (which has a bunch more context so could be worth checking out!)

**Pull request merged**